react/features/toolbox/functions.web.js

```diff
@@ -43,8 +44,10 @@ export function isToolboxVisible(state: Object) {
         visible
     } = state['features/toolbox'];
     const { audioSettingsVisible, videoSettingsVisible } = state['features/settings'];
+    const { isOpen } = state['features/chat'];
+    const isMobileChatOpen = isMobileBrowser() && isOpen;
 
-    return Boolean(!iAmSipGateway && (timeoutID || visible || alwaysVisible
+    return Boolean(!isMobileChatOpen && !iAmSipGateway && (timeoutID || visible || alwaysVisible
```

Comment:
Why do we check if the browser is mobile here?

Reply:
@saghul raised a concern regarding the use case of the conference in a mobile browser, and this is to ensure that it doesn't look botched in a mobile browser.

Reply:
So no toolbar in mobile browsers if you have the chat opened?

Reply:
This was proposed as a solution by @vp8x8 in order to not clutter the UI on the mobile browser